opencti-platform/opencti-front/src/components/FilterIconButton.tsx

```diff
@@ -8,8 +8,8 @@ import { handleFilterHelpers } from '../utils/hooks/useLocalStorage';
 import { filterValuesContentQuery } from './FilterValuesContent';
 import { FilterValuesContentQuery } from './__generated__/FilterValuesContentQuery.graphql';
 
-interface FilterIconButtonProps {
-  availableFilterKeys?: string[];
+export interface FilterIconButtonProps {
+  availableFilterKeys?: string[] | undefined;
```

Comment:
It's not useful to add `| undefined` since you already have `?`

Reply:
Fixed